en/docs/administer/key-managers/configure-global-key-manager.md

```diff
@@ -0,0 +1,124 @@
+# Configure the Global Key Manager
+
+This document outlines the steps to enable the global key manager support in WSO2 API Manager (API-M). With this feature, a single token generated using the global key manager can be used to invoke cross-tenant APIs without generating a token in that specific tenant's key manager.
+
+!!! Note
+      This feature is available only as an update and is available from Update level 4.2.0.82 (released on 13th May 2024).
+
+The global key manager can be created through the admin portal of the super tenant. This global key manager will be visible in the devportal of all the tenants and this can be used to generate a token which can be used to invoke APIs across tenants. Any third party key manager can be configured as a global key manager.
+
+!!! Note
+      It is only possible to create one global key manager which can only be done through the Admin portal of the super tenant.
```

Comment:
Shall we change `Admin portal` to `Admin Portal` to make it consistent through out the doc?

Reply:
fixed